scripts/joint_trajectory_file_playback.py

```diff
@@ -228,7 +230,13 @@ def find_start_offset(pos):
             cmd, values = self._clean_line(values, joint_names)
             #find allowable time offset for move to start position
             if idx == 0:
+                # Set the initial position to be the current pose
```

Comment:
possibly expand this comment to something closer to 'make sure we move slowly to the starting pose of the trajectory from the current point - user may have moved arm since recording... etc etc etc'